cityofaustin/atd-vz-data
# Add confirmation to delete notes on the Location page and Crash page

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/14771

This PR adds an existing `ConfirmModal` component to the `Notes` component so a confirm modal appears when deleting a crash or location note.

## Testing
**URL to test:** 
https://deploy-preview-1339--atd-vze-staging.netlify.app/

**Steps to test:**
1. Log in to the deploy preview with either editor, admin, or IT supervisor permissions
2. Go to a crash page like [this one](https://deploy-preview-1339--atd-vze-staging.netlify.app/#/crashes/19329553) and add a note to the crash
3. Click the trash can icon and you should see a confirm modal pop up
4. Click **Cancel** or the **X icon** and the modal should close without deleting the note
5. Try to delete again and, this time, click **Confirm**, and the note should delete
6. Repeat the previous steps on a location page like [this one](https://deploy-preview-1339--atd-vze-staging.netlify.app/editor/#/locations/05186FFEDB)

---
#### Ship list
